### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by vk20july**
sorry for no description inconvenience sir
this is the video of shifting the new post button with respect to scroll to top button
issue: #411 

https://github.com/narayan954/dummygram/assets/71543332/387584ef-77ea-4784-beca-a582f59c4166

### CODE REVIEW

**Comment by narayan954** [comment]
> sorry for no description inconvenience sir this is the video of shifting the new post button with respect to scroll to top button issue: #411
> 
>  VID-20230602-WA0000.1.mp4

it's okay, I'll review this :)

File: src/App.jsx [hunk 1]
```diff
@@ -218,8 +218,9 @@ function App() {
               color="secondary"
               variant="contained"
               sx={buttonStyle}
+              className="button_post"
```
Comment (narayan954) [inline src/App.jsx:221]: class name button_post is not conventional. If we follow BEM convention, we can name it as something like post__newpost__button.

**Review by narayan954 — CHANGES_REQUESTED** [review]
minor change requested

File: src/App.jsx (hunk 1 above)
Comment (vk20july) [reply src/App.jsx:221]: ok let me do this

**Review by narayan954 — APPROVED** [review]
looks good to me!

**Pull request merged**